packages/sdks-tests/src/snippet-tests/api-endpoint.spec.ts

```diff
@@ -35,5 +35,24 @@ test.describe('API Endpoint', () => {
       expect(req!.method()).toBe('GET');
       expect(queryApiInvocations).toBe(0);
     });
+
+    test('records symbol API calls after page load', async ({ page, packageName }) => {
+      test.skip(packageName !== 'gen1-next');
+      // Start monitoring network requests
+      const urlMatchForContentAPI = /https:\/\/cdn\.builder\.io\/api\/v3\/content\/symbol/;
+
+      const responsePromise = page.waitForResponse(urlMatchForContentAPI);
+
+      await page.goto(
+        '/contentwithsymbol?builder.space=ee9f13b4981e489a9a1209887695ef2b&builder.user.permissions=read%2Ccreate%2Cpublish%2CeditDesigns%2CeditLayouts%2CeditLayers%2CeditContentPriority&builder.user.role.name=Designer&builder.user.role.id=creator&builder.cachebust=true&builder.preview=page&builder.noCache=true&builder.allowTextEdit=true&__builder_editing__=true&builder.overrides.page=6a2f7e1a9ab2454ba6766522d6d99f0d&builder.overrides.6a2f7e1a9ab2454ba6766522d6d99f0d=6a2f7e1a9ab2454ba6766522d6d99f0d&builder.overrides.page:/contentwithsymbol=6a2f7e1a9ab2454ba6766522d6d99f0d&builder.frameEditing=page&builder.options.locale=Default',
+        { waitUntil: 'networkidle' }
```

Comment:
We should avoid `waitUntil` unless absolutely necessary. I don't think we need it in this case.